### DISCUSSION TITLE
Whether skywalking supports monitoring of WeChat Mini Programs

### DISCUSSION THREAD

**Discussion by cuixu3659271** [apache/skywalking#8823] [Q&A] [answered]
Whether skywalking supports monitoring of WeChat Mini Programs?
Whether the WeChat Mini Program uni-appp development can support front-end alarm monitoring

**Comment by wu-sheng** [comment]
I think the simple answer is not yet. @Jtrust may find some time to do. Open-source is open-source, we need someone to take actions.

**Comment by Jtrust** [comment, marked as answer]
I’am going to try to complete the probe SDK.  But it may take some time.
